adm_program/system/classes/phpini.php

```diff
@@ -0,0 +1,201 @@
+<?php
+/**
+ ***********************************************************************************************
+ * Class manages PHP-Ini stuff
+ *
+ * @copyright 2004-2017 The Admidio Team
+ * @see https://www.admidio.org/
+ * @license https://www.gnu.org/licenses/gpl-2.0.html GNU General Public License v2.0 only
+ ***********************************************************************************************
+ */
+
+/**
+ * @class PhpIni
+ */
+class PhpIni
+{
+    const BYTES_UNIT_FACTOR_1024 = 1024;
+    const BYTES_UNIT_FACTOR_1000 = 1000;
+
+    /**
+     * @param string $data
+     * @param int    $multi Factor to multiply. Default: 1024
+     * @return int
+     */
+    private static function getBytesFromSize($data, $multi = self::BYTES_UNIT_FACTOR_1024)
```

Comment:
Please add an description of the method. There should be a description of the expected format within $data